auth/facebooklogin.js

```diff
@@ -0,0 +1,92 @@
+
+var FacebookStrategy = require('passport-facebook').Strategy;
+const session = require('express-session')
+// load up the user model
+var User = require('../model/usermodel');
+
+// load the auth variables
+var config = require('../config');
+
+module.exports = function(app,passport) {
+
+    app.use(session({ secret: 'bnbisgood' })); // session secret
+    app.use(passport.initialize());
+    app.use(passport.session()); // persistent login sessions
+    // used to serialize the user for the session
```

Comment:
Lines 12, 13, 14 to be included in server.js, not here

Reply:
i though it would be cleaner to include in this file itself

Reply:
It's actually not clean. This file should ONLY have the code executed by passport. The configurations should all go in server.js. In fact, the part of the code which has facebook token strategy could also go into a separate file. I'll do it myself